Containerfile

```diff
@@ -0,0 +1,42 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest as BUILDER
+
+ENV OCS_CI_DIR=/opt/ocs-ci
+WORKDIR "${OCS_CI_DIR}"
+
+RUN dnf install --nodocs -y python38 python38-devel git libcurl-devel gcc openssl-devel libxml2-devel
+
+# Copy the entire source tree to the image
+# TODO: check to see if there are any more files we can drop from the image
+# TODO: the azure package is very big, we should consider to see how we can compact it
+COPY / "${OCS_CI_DIR}"
+
+RUN pushd "${OCS_CI_DIR}" \
+    && python3 -m venv venv \
+    && source venv/bin/activate \
+    && pip3 install --upgrade pip \
+    && pip3 install setuptools==65.5.0 \
+    && pip3 install -r requirements.txt \
+    && rm -rf .git
+
+# Part 2
+
+FROM registry.access.redhat.com/ubi8/ubi:latest as RUNNER
+
+ENV OCS_CI_DIR="/opt/ocs-ci" \
+    VIRTUAL_ENV="/opt/ocs-ci/venv" \
+    PATH="/opt/ocs-ci/venv/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+
+WORKDIR "${OCS_CI_DIR}"
+
+RUN curl https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz | tar -C /usr/local/bin -zxvf - oc
+
+RUN dnf install -y --nodocs python38 git jq rsync \
+    && dnf clean all \
+    && rm -rf /var/cache/yum \
+    && curl -sL https://github.com/mikefarah/yq/releases/download/v4.32.2/yq_linux_amd64.tar.gz | tar -C /usr/local/bin -zxvf - ./yq_linux_amd64 \
+    && mv -v /usr/local/bin/yq_linux_amd64 /usr/local/bin/yq
+
+COPY --from=BUILDER "${OCS_CI_DIR}" "${OCS_CI_DIR}"
+COPY --chmod=755 /Docker_files/ocsci_container/scripts/run-ocs-ci.sh /usr/local/bin/
+
+CMD [ "run-ci", "--help" ]
```

Comment:
Can we have `run-ci` as entrypoint?

Reply:
I had this before, and I removed it, as it limit the image execution options